components/grid/grid.html

```diff
@@ -0,0 +1,101 @@
+      <nav id="navbar">
+        <h1 class="logo">
+          <span class="text-primary">
+            <i class="fa-brands fa-uikit"></i>
+            CRIPSY </span
+          >UI
+        </h1>
+        <ul>
+          <li><a href="#">DOCS</a></li>
+          <li><a href="#">GITHUB</a></li>
+        </ul>
+      </nav>
+      <!-- bottom sidebar starts here -->
+      <div class="bottom-nav">
+        <ul>
+          <li><a href="#">Alert</a></li>
+          <li><a href="#">Avatar</a></li>
+          <li><a href="#">Badges</a></li>
+          <li><a href="#">Buttons</a></li>
+          <li><a href="#">Card</a></li>
+          <li><a href="#">Images</a></li>
+          <li><a href="#">Input</a></li>
+          <li><a href="#">Toast</a></li>
+          <li><a href="#">Grids</a></li>
+          <li><a href="#">Navbar</a></li>
+        </ul>
+      </div>
+      <div class="section">
+        <div class="container">
+          <div class="content-section">
+            <div class="title">
+              <h1>Grids</h1>
+            </div>
+            <div class="content">
+              <h3>
+                The grid CSS property is a shorthand property that sets all of the explicit and implicit grid properties in a single declaration.
+              </h3>
+            </div>
+                <h2>Single Column Grid</h1>
+            </div>
+            <div class="grid">
+              <div class="grid-container col-1">
+                <div class="primarybg">Single Column Grid</div>
+              </div>
+        </div>
+        <div class="snippet-container">
+        <pre class="snippet">
+          <code class="language-html">&#x3C;div class=&#x22;grid&#x22;&#x3E;
+              &#x3C;div class=&#x22;grid-container col-1&#x22;&#x3E;
+              &#x3C;div class=&#x22;primarybg&#x22;&#x3E;Single Column Grid&#x3C;/div&#x3E;
+          &#x3C;/div&#x3E;</code></div>
+        </pre>
+            <h2>Two Columns Grid</h2>
+            <div class="grid">
+            <div class="grid-container col-2">
+              <div class="primarybg">Two Columns Grid</div>
+              <div class="secondarybg">Two Columns Grid</div>
+            </div> 
+            <div class="snippet-container">
+              <pre class="snippet">
+                <code class="language-html">&#x3C;div class=&#x22;grid&#x22;&#x3E;
+                &#x3C;div class=&#x22;grid-container col-2&#x22;&#x3E;
+                &#x3C;div class=&#x22;primarybg&#x22;&#x3E;Two Columns Grid&#x3C;/div&#x3E;
+                &#x3C;div class=&#x22;secondarybg&#x22;&#x3E;Two Columns Grid&#x3C;/div&#x3E;
+                &#x3C;/div&#x3E;</code></div>
+              </pre>
+           </div>
+           <h2>Three Columns Grid</h2>
+
+           <div class="grid-container col-3">
+            <div class="primarybg">Three Columns Grid</div>
+            <div class="secondarybg">Three Columns Grid</div>
+            <div class="warningbg">Three Columns Grid</div>
+          </div>
```

Comment:
Three Columns are not distinguishable you can use different color to distinguish them or use border color to distinguish them.